flow360/component/simulation/validation/validation_simulation_params.py

```diff
@@ -39,6 +50,71 @@ def _check_consistency_wall_function_and_surface_output(v):
     return v
 
 
+def _check_cht_solver_settings(params):
```

Comment:
Let's add a docstring here indicating what this function is trying to replace in the casePipeline validation (checkCHTSolverSettings()).